src/content/docs/cypher/query-clauses/call.md

```diff
@@ -100,20 +99,15 @@ CALL show_tables() RETURN *;
 ```
 Output:
 ```
--------------------------------------------------------
-|id     | name        | type    | comment             |
-|UINT64 | STRING      | STRING  | STIRNG              |
--------------------------------------------------------
-|0      | gf          | RDF     |                     |
--------------------------------------------------------
-|1      | gf_TRIPLES  | REL     |                     |
--------------------------------------------------------
-|2      | gf_RESOURCE | NODE    |                     |
--------------------------------------------------------
-|3      | person      | NODE    | person info         |
--------------------------------------------------------
-|4      | knows       | REL     | person knows person |
--------------------------------------------------------
+┌─────────────┬──────────────┬────────┬─────────────────────────┐
+│ id          │ name         │ type   │ Comment                 │
+│ INT32       │ STRING       │ STRING │ STRING                  │
+├─────────────┼──────────────┼────────┼─────────────────────────┤
+│ 0           │ gf_TRIPLES   │ REL    │                         │
+│ 1           │ gf_RESOURCE  │ NODE   │                         │
+│ 2           │ person       │ NODE   │ person info             │
+│ 3           │ knows        │ REL    │ person knows person     │
+└─────────────┴──────────────┴────────┴─────────────────────────┘
```

Comment:
Could you also rerun the `CALL db_version() RETURN *` on your machine and paste the new CLI output in the section above? 

Let's be ambiguous about the db version number in the output, e.g:
```
┌─────────┐
│ version │
│ STRING  │
├─────────┤
│ 0.x.0   │
└─────────┘
```